### ISSUE TITLE
proton export

### ISSUE THREAD

**Issue by mah11** [geneontology/go-ontology#13271]
From the paper cited for the def, it looks like this does happen normally under some circumstances in fission yeast, so here's a term request, sort of modeled on GO:0097623:

[Term]
id: GO:new
name: proton export across plasma membrane
namespace: biological_process
def: "The directed movement of protons (hydrogen ions) from inside a cell, across the plasma membrane and into the extracellular region." [GOC:mah, PMID:9762918]
synonym: "hydrogen ion export across plasma membrane" EXACT []
synonym: "proton export from cell" EXACT []
is_a: GO:0015992 ! proton transport
intersection_of: GO:0006810 ! transport
intersection_of: exports CHEBI:29103 ! potassium(1+)
intersection_of: has_target_end_location GO:0005576 ! extracellular region
intersection_of: has_target_start_location GO:0005829 ! cytosol
intersection_of: results_in_transport_across GO:0005886 ! plasma membrane

I included "proton export from cell" as a synonym rather than a separate term because I don't know of any proton export that wouldn't go via transmembrane proton-transporting ATPases. Adjust the synonym scope as you see fit.

**Comment by krchristie** [comment]
Hi @mah11 

Thanks for the suggested term you modeled on. That was super helpful!!

Since we already had existing terms that referred to 'hydrogen ion export' in the primary term name, I did that for this one, and put the 'proton export' in the synonyms. I see your point about not making a term when there's no obvious need, so I did include the synonyms for 'export from cell', but thought it might be better to make them BROAD instead of EXACT. So, here's the term I created:

+id: GO:0120029
+name: hydrogen ion export across plasma membrane
+namespace: biological_process
+def: "The directed movement of hydrogen ions (protons) from inside a cell, across the plasma membrane and into the extracellular region." [GOC:mah, PMID:9762918]
+synonym: "hydrogen ion export from cell" BROAD []
+synonym: "proton export across plasma membrane" EXACT []
+synonym: "proton export from cell" BROAD []
+intersection_of: GO:0006810 ! transport
+intersection_of: exports CHEBI:24636 ! proton
+intersection_of: has_target_end_location GO:0005576 ! extracellular region
+intersection_of: has_target_start_location GO:0005829 ! cytosol
+intersection_of: results_in_transport_across GO:0005886 ! plasma membrane
+created_by: kchris
+creation_date: 2017-03-31T17:39:03Z
